src/js/stores/DCOSStore.js

```diff
@@ -26,50 +27,42 @@ class DCOSStore extends EventEmitter {
 
     this.data = {
       marathon: new ServiceTree(),
-      mesos: {
-        frameworks: []
-      },
+      mesos: new SummaryList(),
       dataProcessed: false
     };
 
     this.proxyListeners = [
```

Comment:
Sorry, my comment wasn't very specific. I was wondering if you guys added enough magic, to enable it to automatically register listeners on a different store whenever someone adds a listener to this store 🔮⚛✨